js/buffer.ts

```diff
@@ -170,7 +171,10 @@ export class Buffer implements Reader, Writer {
       // don't spend all our time copying.
       copyBytes(this.buf, this.buf.subarray(this.off));
     } else if (c > MAX_SIZE - c - n) {
-      throw Error("ErrTooLarge"); // TODO DenoError(TooLarge)
+      throw new DenoError(
+        ErrorKind.TooLarge,
+        "The buffer can't grow because it becomes too large"
```

Comment:
This isn't the best English.  "The buffer cannot be grown beyond the maximum size." would be better.